pymer4/models/Lmer.py

```diff
@@ -5,28 +5,29 @@
 Main class to wrap R's lme4 library
 """
 
-
-from copy import copy
-from rpy2.robjects.packages import importr
-import rpy2.robjects as robjects
-from rpy2.rinterface_lib import callbacks
-import rpy2.rinterface as rinterface
-from rpy2.robjects.conversion import localconverter
-from rpy2.robjects import numpy2ri
-import warnings
 import traceback
+import warnings
+from copy import copy
+
+import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
-import matplotlib.pyplot as plt
+import rpy2.rinterface as rinterface
+import rpy2.robjects as robjects
 import seaborn as sns
+from pandas.api.types import CategoricalDtype
```

Comment:
Is this order of import ok to your PyCharm or isort (I'm curious!)

Reply:
isort actually was ok with most of it.

It only changed these orders:

from ..bridge import R2numpy, R2pandas, con2R, pandas2R
from ..utils import (
    _perm_find,
    _return_t,
    _sig_stars,
    _to_ranks_by_group,
)